hadoop-ozone/ozonefs/src/main/java/org/apache/hadoop/fs/ozone/BasicOzoneFileSystem.java

```diff
@@ -113,7 +113,7 @@ public void initialize(URI name, Configuration conf) throws IOException {
     String remaining = matcher.groupCount() == 3 ? matcher.group(3) : null;
 
     String omHost = null;
-    String omPort = String.valueOf(-1);
+    int omPort = -1;
```

Comment:
Minor Nit: We don't need to set it to -1.

Reply:
I think we should initialize it. Because In a corner case where `remaining` is empty, omPort would be uninitialized when reaching line 154 (createAdapter()). I think that might be an issue.

Reply:
Thanks @smengcl for offline explanation why we need to set this.